shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/resource/DataSourceQueryResultSet.java

```diff
@@ -54,22 +61,17 @@
     @Override
     public void init(final ShardingSphereMetaData metaData, final SQLStatement sqlStatement) {
         resource = metaData.getResource();
-        Optional<MetaDataPersistService> persistService = ProxyContext.getInstance().getContextManager().getMetaDataContexts().getMetaDataPersistService();
-        
-        if (persistService.isPresent()) {
-            dataSourcePropsMap = persistService.get().getDataSourceService().load(metaData.getName());
-        } else {
-            dataSourcePropsMap = new LinkedHashMap<>(metaData.getResource().getDataSources().size(), 1);
-            for (Entry<String, DataSource> entry : metaData.getResource().getDataSources().entrySet()) {
-                dataSourcePropsMap.put(entry.getKey(), DataSourcePropertiesCreator.create(entry.getValue()));
-            }
+        dataSourcePropsMap = new LinkedHashMap<>(metaData.getResource().getDataSources().size(), 1);
+        for (Map.Entry<String, DataSource> entry : metaData.getResource().getDataSources().entrySet()) {
```

Comment:
Please use `Entry` instead of `Map.Entry`

Reply:
OK